### ISSUE TITLE
Can't seem to get the OSX reader example working?

### ISSUE THREAD

**Issue by ogroves** [TheLevelUp/ZXingObjC#250]
It's been a fair while since they've been updated - are they working for everyone else?

**Comment by benjohnde** [comment]
Any log / error msg? I may try the OS X reader on Wednesday. Looking forward to give you some feedback then.

**Comment by ogroves** [comment]
Thanks Ben - No errors, the app builds fine but I can't get anything to read.

**Comment by mikecsh** [comment]
Did anyone make any progress here? I can get it to build and show a capture session but no barcode scanning..

**Comment by ogroves** [comment]
I couldn't get it working, it was probably down to my user error more than anything else though. If you get it working I'd appreciate knowing what you did.

**Comment by TongKuo** [comment]
I can't either, same description as Ollie's.

**Comment by mikecsh** [comment]
I also haven't managed to get it working, but I have managed to get it *partially* working in my own app. It will display the capture view and recognise QR codes, but doesn't seem to recognise any 1D barcodes yet.. I'll post some code shortly

**Comment by ogroves** [comment]
Thanks Mike, some progress is good progress.

**Comment by mikecsh** [comment]
Okay - having had a bit of a dig through the code with a debugger, I have got this working in my Mac app now. The key thing is setting `self.capture.hints.tryHarder = YES;` in the initial setup. This is now allowing ZXingObjC to recognise every type of barcode I have to hand as well as the QR codes which it was already recognising.

Being new to this codebase I'm not entirely sure why this should be necessary for it to recognise these barcodes. The flag is used to perform more detailed analysis of each image (i.e. increased accuracy at the expense of speed), but it doesn't *seem* right for it not to be capable of any recognition without this.

I also found it to be quite picky about the lighting of the barcode but this may be subjective.

Code: https://github.com/mikecsh/ZXingObjCTest
